apps/server/src/terrain.ts

```diff
@@ -0,0 +1,49 @@
+import * as fs from 'fs';
+
+import axios from 'axios';
+
+import * as path from 'path';
+import * as os from 'os';
+
+
+const SIMBRIDGE_FOLDER = path.join(os.homedir() + '/flybywire-externaltools-simbridge');
```

Comment:
```suggestion
const SIMBRIDGE_FOLDER = path.join(os.homedir(), 'Documents', 'FlyByWire Simulations', 'SimBridge Resources');
```

`homedir()` alone would save the folder on the route user folder, which is not ideal, recommend using the Documents folder and specifically also creating a FlyByWire Simulations root dir

Reply:
In addition, would recommend using the `getSimbridgeDir` function for this as well to keep things consistent

Reply:
Agree with tracer here, People on Windows 10/11 with active onedrive sync have their documents one subfolder deeper under Onedrive/Documents. ENV Vars def the way to go for that handling